### PR TITLE
[12.0][FIX] eslint on 12.0 branch

### PR DESCRIPTION
**Pull request by legalsylvain**
This trivial patch 
- fixes eslint warning present in the Travis log.
- add a link to the OCA / pos wiki

https://travis-ci.org/OCA/pos/jobs/553725259#L291

Before : 

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,10 @@ pos
 Point of sale
 
 
+Note
+~~~~
+
+Regarding lint warning and errors on Travis you can follow the steps
```
Comment (pedrobaeza) [inline README.md]: Isn't better a link to the general guidelines?

Comment (pedrobaeza) [reply README.md]: In fact, we usually put this file in root: https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/CONTRIBUTING.md

Comment (legalsylvain) [reply README.md]: yes indeed ! but is there a part of guidelines that mentions how to use eslint ? I mean to, automatically fix most of the errors, based on eslint tools and ``.jslintrc`` OCA guidelines, present in pylint-odoo.

**Review by KolushovAlexandr — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pedrobaeza) [reply README.md]: If not, we should add it :wink:

Comment (legalsylvain) [reply README.md]: Hi. 
- I removed the last commit
- added the contributing.md file.
- proposed some text in the OCA guidelines (https://github.com/OCA/odoo-community.org/pull/43)

Is it all ok ? 

Thanks ! 

regards.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**